wd_helper.au3

```diff
@@ -1343,6 +1343,9 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 			If $iErr = $_WD_ERROR_Success Then
 				Local $bDriverExists = ($sDriverCurrent <> 'None')
 
+				; Determine if the remote and local files are the same size, just in case they are different, force a download. This is needed because 32/64 bit versions have different file sizes.
+				If FileGetSize($sInstallDir & $sDriverEXE) <> InetGetSize($sURLNewDriver, $INET_FORCERELOAD) Then $bForce = True
+
```

Comment:
This is required due to the fact that `_WinAPI_GetBinaryType()` returns `$SCS_DOS_BINARY` and you can not determine if current version is 32/64bit because `msedgedriver.exe --version` and also `chromedriver.exe --version` returns the same information for `--version` for both 32 and 64 bit versions..

@Danp2  waiting for yours approval.
I think that this will be all in this PR

Reply:
> This is required due to the fact that _WinAPI_GetBinaryType() returns $SCS_DOS_BINARY

Are you sure about this? I got $SCS_32BIT_BINARY and $SCS_64BIT_BINARY when I tested it.

Reply:
Maybe a difference between Win7 and Win10?

Reply:
using this code:

```AutoIt
#include <WinAPIFiles.au3>
ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\chromedriver.exe") & @CRLF)
ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\geckodriver.exe") & @CRLF)
ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\msedgedriver.exe") & @CRLF)

ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\1\chromedriver.exe") & @CRLF)
ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\1\geckodriver.exe") & @CRLF)
ConsoleWrite("- " & @ScriptLineNumber & " " & _WinAPI_GetBinaryType("z:\Test\WebDriver\1\msedgedriver.exe") & @CRLF)
Exit

```
I always get `1`

Reply:
I use Windows 11 on my notebook.

Reply:
Take a look here:
https://www.autoitscript.com/forum/topic/205553-webdriver-udf-help-support-iii/?do=findComment&comment=1495308

Reply:
Done